### DISCUSSION TITLE
GitHub Android version still logged in after my password reset

### DISCUSSION THREAD

**Discussion by huangyxi** [community/community#12126] [Mobile] [answered]
I reset my password on GitHub on the web and the other devices I've logged in before to are automatically logged out. However, Android GitHub (version 1.53.0) is still logged in and has access to my private repositories.

**Comment by stevepopovich** [comment]
Hi! Thanks for logging this issue. We will look into this. Thanks!

**Comment by stevepopovich** [comment, marked as answer]
Hi Mark,

We have chatted with the team about this and have determined that this is current intended behavior. 

Basically, mobile log-in sessions are actually managed by their own separate OAuth tokens and are not tied directly to your web session. This means changing your password does not automatically revoke all your OAuth tokens. 

We have updated docs [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/updating-your-github-access-credentials#updating-your-access-tokens) with instructions how to update your OAuth tokens to sign out of your mobile sessions. 

We are investigating whether the change password flow should invalidate mobile tokens and this may change in the future. Thanks for writing in.

**Reply by huangyxi** [reply]
Minor security concern in an imagined situation: if someone lost their phone, they might just reset their password to protect their account, as traditional web apps tell them to do. 
But in most cases, I think it's OK if the mobile app signs out automatically or not, and thanks for the feedback.
